libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -268,51 +268,47 @@ public boolean isShowingInstructionList() {
   }
 
   /**
-   * Hide the instruction list and show the sound button.
+   * Hide the instruction list.
    * <p>
    * This is based on orientation so the different layouts (for portrait vs. landscape)
    * can be animated appropriately.
    */
   public void hideInstructionList() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
This block of code is duplicated 👀 https://github.com/mapbox/mapbox-navigation-android/blob/ca5bcc5bcb1b535035ae7f85bddf1b2dbced52db/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java#L298-L300
Could we extract it into a method and call it from here and from `showInstructionList()`?

Reply:
We could also [decompose these conditionals](https://sourcemaking.com/refactoring/decompose-conditional), you know, we should keep `public` methods as clean as possible so they read like pseudocode (i.e. no conditionals, `for`s, etc.).